### ISSUE TITLE
Query parameter is double encoded

### ISSUE THREAD

**Issue by novi** [ishkawa/APIKit#59]
A query parameter with GET method is double encoded.  It converts "あ" into "%25E3%2581%2582", it should be "%E3%81%82".

`query` property of NSURLComponents will be encoded automatically when getting to complete NSURL. You can use `NSURLComponents.percentEncodedQuery` or just use NSURLComponents and NSURLQueryItem for query encoding.

**Comment by ishkawa** [comment]
Thank you for reporting! Could you check if this issue is fixed in @a3d9621?

**Comment by novi** [comment]
Thanks! It is fine for iOS. But unable to build for Mac since NSURLQueryItem needs OS X 10.10.
(Sorry, I have mentioned NSURLQueryItem has no support for OS X 10.9.)

```
APIKit/APIKit/Request.swift:84:28: error: 'NSURLQueryItem' is only available on OS X 10.10 or newer
```

**Comment by ishkawa** [comment]
Ah, I've completely forgotten! This may be fixed in @d989ce8.
Please review #60 again.

**Comment by novi** [comment]
Both the platform is Okay. Thank you for fixing.
